### ISSUE TITLE
Forcefully add some classes (like sap/ui/core/ComponentSupport) to a self-contained build

### ISSUE THREAD

**Issue by kristian** [SAP/ui5-tooling#381]
## Expected Behavior
Only the sap-ui-custom.js boostrap JavaScript should be loaded when a self-contained build is chosen. The boostrap should contain all classes needed to run the application.

## Current Behavior
Some files are anyways loaded seperately, such as the sap/ui/core/ComponentSupport in case the data-sap-ui-oninit boostrap option is chosen to load a component.

## Steps to reproduce the issue
1. Change the index.html file of the samle application to load the ComponentContainer via the data-sap-ui-oninit="module:sap/ui/core/ComponentSupport" method
2. Build & serve the sample app self-contained

## Context
- OS/Platform: Windows
- Node.js Version: 10.11.0
- npm Version: 6.4.1
- Browser *(if relevant)*: Chrome Dev.
- Other information: Tested on 1.59.0

## Affected components
- [x] [ui5-builder](https://github.com/SAP/ui5-builder)
- [ ] [ui5-server](https://github.com/SAP/ui5-server)
- [ ] [ui5-cli](https://github.com/SAP/ui5-cli)
- [ ] [ui5-fs](https://github.com/SAP/ui5-fs)
- [ ] [ui5-project](https://github.com/SAP/ui5-project)
- [ ] [ui5-logger](https://github.com/SAP/ui5-logger)

**Comment by henriquemattos** [comment]
What would be the status of this regarding TypeScript? Is this required to be imported if it's not explicitly used?

**Comment by matz3** [comment]
Yes, for TypeScript there shouldn't be any difference apart from how the module dependency is defined (`sap.ui.define` vs `import`).

**Comment by henriquemattos** [comment]
Hi, @matz3 . Thank you for answering. These "community-driven" projects are sometimes complex, and makes it overwhelming. In the end I preferred to go via the [Walkthrough Tutorial Typescript](https://ui5.sap.com/#/topic/dad1905a07f849ce9c509721317d38d8) in the UI5 documentation which took me in a very simple project structure without multiple dev-dependencies.

I appreciate your time and your support.

Kind regards,
Henrique

**Comment by henriquemattos** [comment]
Hi, @matz3 . I'm sorry for the misleading comment. At the time I was working with ui5-tooling and ui5-easy-generator + generator-ui5-project, trying to bootstrap a TypeScript skeleton. I probably came across this ComponentSupport issue with one of the tutorials, and I commented on the wrong issue.

Should I delete my comment (if this is possible)?

**Comment by matz3** [comment]
Ah, I understand. I don't see a reason to delete the comment, but I could do so if you want to (and maybe you can even do it by yourself).

The TypeScript Walkthrough does not mention this workaround as it does not focus on a self-contained build.
